src/edit/command.js

```diff
@@ -102,7 +102,7 @@ export class CommandSet {
     this.op = op
   }
 
-  // :: (union<Object<CommandSpec>, string>, ?(string, CommandSpec) → bool) → CommandSet
+  // :: (union<Object<CommandSpec>, "schema">, ?(string, CommandSpec) → bool) → CommandSet
```

Comment:
String (or number) literals aren't something that getdocs currently parses as a type, so putting them in these signatures would break the doc build.

I guess these could be treated as named typed with the literal syntax as their name without causing much harm. Does that sound right?